docs/Installation-Windows.md

```diff
@@ -153,6 +153,12 @@ pip install -e .
 This will complete the installation of all the required Python packages to run
 the ML-Agents toolkit.
 
+If you are stuck during this installation, you can try:
```

Comment:
Maybe adding a little more description of what being "stuck" is like and add a few lines describing what `--no-cache-dir` does.

Reply:
Will change.